Fix msi after datadog to otel rename

dszmigielski opened the pull request:
Fixed msi after rebranding to otel:

- company name in default path was fixed
- logos were exchanged with opentelemetry ones
- msi filename was fixed

pellared commented:
You have removed two images but it looks they are still used here: https://github.com/dszmigielski/opentelemetry-dotnet-instrumentation/blob/fix-msi-after-datadog-to-otel-rename/src/WindowsInstaller/Product.wxs#L29-L30

dszmigielski commented:
> You have removed two images but it looks they are still used here: https://github.com/dszmigielski/opentelemetry-dotnet-instrumentation/blob/fix-msi-after-datadog-to-otel-rename/src/WindowsInstaller/Product.wxs#L29-L30

I didn't remove them, I changed the content without renaming them. The only one that required renaming was the icon one, and I changed the reference in project. But yep - I missed the Product.wsx change there for icon reference (Rider didn't find  that when I applied renaming).

pjanotti reviewed (approved):
LGTM
